cluster/src/main/scala/org/apache/spark/executor/SnappyExecutor.scala

```diff
@@ -87,6 +87,10 @@ class SnappyExecutor(
         if (!ContextJarUtils.checkItemExists(ContextJarUtils.droppedFunctionsKey, appName)) {
           logInfo(s"Creating ClassLoader for $appName" +
               s" with dependencies $appDependencies")
+          var includeInGlobalCmdRegion = true
```

Comment:
Can be shortened as 
`val includeInGlobalCmdRegion = !appDependencies.contains(io.snappydata.Constant.SNAPPY_JOB_URL)`